### PR TITLE
Add Recipe for pysis

### PR DESCRIPTION
**Pull request by Kelvinrr**
[pysis](https://github.com/wtolson/pysis) is a simple wrapper for USGS-Astrogeology's [ISIS3](https://github.com/USGS-Astrogeology/ISIS3) applications. 

Seems good to go for review @conda-forge/help-python

### CODE REVIEW

File: recipes/pysis/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    - bash tests/make_isis_env.sh
+    - python setup.py install
+
+requirements:
+  build:
+    - python
+    - numpy
+    - six
+    - pvl
+    - pytest
```
Comment (marcelotrevisani) [inline recipes/pysis/meta.yaml]: Do you really need ``pytest`` to build this package? or just to test this library?

Comment (Kelvinrr) [reply recipes/pysis/meta.yaml]: Just for testing, this release has already been tested using CI so this line can just go.

Comment (marcelotrevisani) [reply recipes/pysis/meta.yaml]: Please remove ``pytest`` then

File: recipes/pysis/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+build:
+  skip: True  # [win]
+  number: 0
+  script:
+    - bash tests/make_isis_env.sh
+    - python setup.py install
+
+requirements:
+  build:
+    - python
```
Comment (marcelotrevisani) [inline recipes/pysis/meta.yaml]: Please add ``setuptools``

**Pull request merged**